db/schema.rb

```diff
@@ -10,11 +10,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_01_15_215627) do
+ActiveRecord::Schema.define(version: 2019_01_16_152908) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "active_storage_attachments", force: :cascade do |t|
```

Comment:
Why is active storage updated in this PR?

Reply:
this was a #&%!up so i deleted the schema file and re ran bundle and db:migrate